eo-maven-plugin/src/main/resources/org/eolang/maven/transpile/to-java.xsl

```diff
@@ -171,6 +171,11 @@
       </xsl:otherwise>
     </xsl:choose>
   </xsl:function>
+  <!-- Escape `+` in test name syntax. -->
+  <xsl:function name="eo:escape-plus">
+    <xsl:param name="name"/>
+    <xsl:value-of select="replace($name, '\+', '')"/>
```

Comment:
@h1alexbel let's replace only the first `+` char instead of replacing all of them